emmet: extra packaging steps may be required

Based on the upstream (theia) repo, there looks to be a problem with `emmet` and one of it's dependencies (`vscode-emmet-helper`) https://github.com/eclipse-theia/theia/issues/7019.

The builtin complains about the following when attempting to perform IntelliSense:

```
Uncaught Error: Cannot find module 'vscode-emmet-helper'

Error: Cannot find module 'vscode-emmet-helper'
    at Function.Module._resolveFilename (:3000/internal/modules/cjs/loader.js:582)
    at Function.Module._load (:3000/internal/modules/cjs/loader.js:508)
    at Function.module._load (:3000/Users/vincentfugnitto/workspace/theia/packages/plugin-ext-vscode/lib/node/plugin-vscode-init.js:80)
    at Module.require (:3000/internal/modules/cjs/loader.js:637)
    at require (:3000/internal/modules/cjs/helpers.js:22)
    at Object.<anonymous> (:3000/Users/vincentfugnitto/workspace/theia/plugins/vscode-builtin-emmet/extension/dist/extension.js:1)
    at n (:3000/Users/vincentfugnitto/workspace/theia/plugins/vscode-builtin-emmet/extension/dist/extension.js:1)
    at Object.t.getEmmetHelper (:3000/Users/vincentfugnitto/workspace/theia/plugins/vscode-builtin-emmet/extension/dist/extension.js:1)
    at t.DefaultCompletionItemProvider.provideCompletionItemsInternal (:3000/Users/vincentfugnitto/workspace/theia/plugins/vscode-builtin-emmet/extension/dist/extension.js:1)
    at t.DefaultCompletionItemProvider.provideCompletionItems (:3000/Users/vincentfugnitto/workspace/theia/plugins/vscode-builtin-emmet/extension/dist/extension.js:1)
    at errors.ts:26
```

I'm thinking it might be a similar issue as `typescript` where we had to perform some manual steps to package correctly.

--- a/src/vscode-bundle.js
+++ b/src/vscode-bundle.js
@@ -3,7 +3,7 @@ const fs = require('fs');
 const rimraf = require('../vscode/node_modules/rimraf');
 const vfs = require('../vscode/node_modules/vinyl-fs');
 const ext = require('../vscode/build/lib/extensions');
-const { root, theiaExtension, extensions } = require('./paths.js')
+const { root, theiaExtension, extensions, run } = require('./paths.js')
 
 const backupNodeModules = () => {
     const move = (src, dest) => fs.existsSync(src) && fs.renameSync(src, dest);
@@ -23,4 +23,5 @@ rimraf.sync(extensions());
         stream.on('error', reject);
         stream.on('end', resolve);
     });
+    await run('yarn', ['install', '--production'], extensions('emmet'));
 })();

